client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-definitions/post/index.tsx

```diff
@@ -15,8 +15,8 @@ const getFirstPostPublished: TaskAction = ( task, flow, context ): Task => {
 			( task.completed && isBlogOnboardingFlow( flow || null ) ) ||
 			false,
 		calypso_path: ! isBlogOnboardingFlow( flow || null )
-			? `/post/${ siteInfoQueryArgs?.siteSlug }`
-			: addQueryArgs( `https://${ siteInfoQueryArgs?.siteSlug }/wp-admin/post-new.php`, {
+			? `/post/${ siteSlug }`
```

Comment:
It was using siteSlug from the siteInfoQueryArgs when the siteSlug was already available, so I used it.